data/weapons/weapon_data.csv

```diff
@@ -1,59 +1,59 @@
+Thunderbolt MRM Rack,diableavionics_thunderboltsmall,0,,200,1800,,800,,5,6,6,8,,,HIGH_EXPLOSIVE,0,,0,5.5,2,0.5,6,9,9,9,,400,50,10,250,,,DO_NOT_AIM,"missile7, strike4, diable_weapon_bp",,Diable Avionics,,Finisher,Medium,Good,,,"%s: spreads up to %s of its damage inside the enemy ship, disabling weapons away from the edge of the hull.",Pierce Damage|half,,,TRUE,11.4
+Thunderbolt MRM Pod,diableavionics_thunderboltpod,1,,1150,2500,,800,,5,6,14,24,,,HIGH_EXPLOSIVE,0,,0.4,6,4,0.4,0,12,4,12,,400,650,10,250,,,DO_NOT_AIM,"missile13, strike9, diable_weapon_bp, rare_bp",,Diable Avionics,,Finisher,Medium,Good,,,"%s: spreads up to %s of its damage inside the enemy ship, disabling weapons away from the edge of the hull.",Pierce Damage|half,,,TRUE,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Banish Torpedo (twin),diableavionics_banishsmall,1,,350,1250,,2000,2000,500,18,2,2,,,ENERGY,0,,0,10,1,,0,0,0,0,,10,120,6,300,,,"STRIKE, USE_VS_FRIGATES, DO_NOT_AIM","strike6, missile1, diable_weapon_bp",,Diable Avionics,,,Fast,Pre-Aimed,,,,,,,TRUE,9.7
-Banish Torpedo Launcher,diableavionics_banish,2,0.75,1550,1500,,2000,2000,500,25,11,6,,,ENERGY,500,,0,10,1,,0,0,0,0,,10,120,6,300,,,"STRIKE, USE_VS_FRIGATES, DO_NOT_CONSERVE, DO_NOT_AIM","missile13, strike9, diable_weapon_bp",,Diable Avionics,,Opener,Fast,Pre-Aimed,,,,,,,TRUE,
+Banish Torpedo (twin),diableavionics_banishsmall,1,,350,1250,,3000,3000,500,18,2,2,,,ENERGY,0,,0,10,1,,0,0,0,0,,10,120,6,300,,,"STRIKE, USE_VS_FRIGATES, DO_NOT_AIM","strike6, missile1, diable_weapon_bp",,Diable Avionics,,,Fast,Pre-Aimed,,,,,,,TRUE,9.7
+Banish Torpedo Launcher,diableavionics_banish,2,0.75,1550,1500,,3000,3000,500,25,11,6,,,ENERGY,0,,0,10,1,,0,0,0,0,,10,120,6,300,,,"STRIKE, USE_VS_FRIGATES, DO_NOT_CONSERVE, DO_NOT_AIM","missile13, strike9, diable_weapon_bp",,Diable Avionics,,Opener,Fast,Pre-Aimed,,,,,,,TRUE,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 Plover Rocket Pod,diableavionics_plover,0,,1220,1250,,600,,33,12,10,50,,,HIGH_EXPLOSIVE,0,,0,4.05,4,0.15,0,1,0.1,1,,500,50,4,100,,,,"rocket13, strike8, diable_weapon_bp",,Diable Avionics,,Anti Armor,Medium,None,,,"%s: spreads up to %s of its damage inside the enemy ship, disabling weapons away from the edge of the hull.",Pierce Damage|half,,,,
-Thrush LRM System,diableavionics_thrush,1,,1150,7000,,700,,65,30,13,10,0.06667,2,HIGH_EXPLOSIVE,0,,0,10,2,1,0,0,0,0,,425,25,16,150,,,"DO_NOT_AIM, CONSERVE_1","missile12, strike8, diable_weapon_bp, rare_bp",,Diable Avionics,,Stand-off Support,Medium,Average,,,%s: The missiles deploy %s if they miss their intended target.,Secondary Payload|4 Micro-missiles,,,,
+Thrush LRM System,diableavionics_thrush,1,,1150,7000,,1000,,65,30,13,10,1.3334,2,HIGH_EXPLOSIVE,0,,0,9,2,1,0,0,0,0,,425,25,16,300,,,"DO_NOT_AIM, CONSERVE_1","missile12, strike8, diable_weapon_bp, rare_bp",,Diable Avionics,,Stand-off Support,Medium,Average,,,%s: The missiles deploy %s if they miss their intended target.,Secondary Payload|12 Micro-missiles,,,,
```

Comment:
Thrush LRM System damage increased to 1000 (from 700). Ammo regeneration rate increased to 1.3334 per second (from 0.06667 per second. pretty sure that's a 20x increase.) Projectile health increased to 300 (from 150). Micro-missile payload increased to 12 (from 4).